<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1380</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>687</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="drawCube">
            <property name="minimumSize">
             <size>
              <width>99</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>99</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>Show Cube</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="drawGridX">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-color: rgb(0, 249, 0);</string>
            </property>
            <property name="text">
             <string>Show Grid X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="drawGridY">
            <property name="minimumSize">
             <size>
              <width>99</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>99</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">selection-background-color: rgb(0, 249, 0);</string>
            </property>
            <property name="text">
             <string>Show Grid Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="drawTypeComboBox">
            <property name="minimumSize">
             <size>
              <width>141</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>141</width>
              <height>27</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Wireframe</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Solid</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Points</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>101</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>Set Single Step:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="SingleStep_2">
            <property name="minimumSize">
             <size>
              <width>141</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>141</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>0.500000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>91</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>91</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Set Bounds:</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="Bounds">
            <property name="minimumSize">
             <size>
              <width>141</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>141</width>
              <height>31</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>5.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>111</width>
              <height>31</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>111</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>Choose color:</string>
            </property>
            <property name="textFormat">
             <enum>Qt::RichText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="CurrentColor">
            <property name="minimumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QLabel" name="functionLabel">
            <property name="minimumSize">
             <size>
              <width>56</width>
              <height>21</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>56</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Function:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="functionLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>2</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>169</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <property name="placeholderText">
             <string>Enter function</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="deleteSelected">
            <property name="minimumSize">
             <size>
              <width>131</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>131</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Monserat&quot;;
color: rgb(0, 0, 0);
border-radius: 10px;
padding: 5px;</string>
            </property>
            <property name="text">
             <string>delete selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Intersections">
            <property name="minimumSize">
             <size>
              <width>131</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>131</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);
font: 14pt &quot;Monserat&quot;;
color: rgb(0, 0, 0);
border-radius: 10px;
padding: 5px;</string>
            </property>
            <property name="text">
             <string>Show Intersectons</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="sizeConstraint">
           <enum>QLayout::SetFixedSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="SaveToFile">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>35</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>32</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 6px;</string>
            </property>
            <property name="text">
             <string>Save to File</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="Table"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-radius: 10px;
padding: 5px;</string>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>900</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1380</width>
     <height>36</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
